addons/web/static/src/js/views/search_panel.js

```diff
@@ -58,13 +69,34 @@ function _processSearchPanelNode(node, fields) {
         if (section.type === 'category') {
             section.icon = section.icon || 'fa-folder';
             section.hierarchize = !!pyUtils.py_eval(attrs.hierarchize || '1');
+            if (section.enableCounters) {
+                incompatibilities.category = childNode;
+            }
```

Comment:
no need

Reply:
Why? The condition to disable counters is if there is a filter with a domain and a category with counters, right?